lib/business/calendar.rb

```diff
@@ -25,9 +25,8 @@ def self.load(calendar_name)
       end
 
       new(
-        holidays: data["holidays"],
-        working_days: data["working_days"],
-        extra_working_dates: data["extra_working_dates"],
+        calendar_name,
+        **data.slice("holidays", "working_days", "extra_working_days"),
```

Comment:
I think we should convert `initialize` to take kwargs here and keep the explicit definition of the arguments.

```suggestion
        name: calendar_name,
```